admin/controllers/service-manager.js

```diff
@@ -104,7 +104,7 @@ const setUpdateTiming = async (req, res, next) => {
   }
 
   try {
-    await settingService.update(settings.loadingTimeLimit, settings.questionTimeLimit, req.session.userId)
+    await settingService.update(settings.loadingTimeLimit, settings.questionTimeLimit, req.session.passport.user.id)
```

Comment:
We would usually write this as `req.user.id` rather than access the session / passport data directly.

Reply:
done 👍